testdata/picard/analysis/artifacts/CollectSequencingArtifactMetrics/ExpectedMetricsOutput/with_context.pre_adapter_detail_metrics

```diff
@@ -1,201 +1,201 @@
 ## htsjdk.samtools.metrics.StringHeader
-# picard.analysis.artifacts.CollectSequencingArtifactMetrics MINIMUM_INSERT_SIZE=30 MAXIMUM_INSERT_SIZE=30 CONTEXT_SIZE=1 INPUT=/Users/msooknah/Documents/gp_projects/picard-private/Picard-public/testdata/picard/analysis/CollectSequencingArtifactMetrics/test.sam OUTPUT=/Users/msooknah/Documents/scratch/picardgroup/oxog/unit/tmp_output/with_context REFERENCE_SEQUENCE=/Users/msooknah/Documents/gp_projects/picard-private/Picard-public/testdata/picard/analysis/CollectSequencingArtifactMetrics/test.fasta    MINIMUM_QUALITY_SCORE=20 MINIMUM_MAPPING_QUALITY=30 INCLUDE_UNPAIRED=false TANDEM_READS=false USE_OQ=true ASSUME_SORTED=true STOP_AFTER=0 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false
+# picard.analysis.artifacts.CollectSequencingArtifactMetrics MINIMUM_INSERT_SIZE=30 MAXIMUM_INSERT_SIZE=30 CONTEXT_SIZE=1 INPUT=/Users/farjoun/picard-private/Picard-public/testdata/picard/analysis/artifacts/CollectSequencingArtifactMetrics/test.sam OUTPUT=/var/folders/tc/hy9lszxd1dg9cf4bky51mrrd9k3s6g/T/artifactMetrics.7914109507701238908.tmp/with_context REFERENCE_SEQUENCE=/Users/farjoun/picard-private/Picard-public/testdata/picard/analysis/artifacts/CollectSequencingArtifactMetrics/test.fasta    MINIMUM_QUALITY_SCORE=20 MINIMUM_MAPPING_QUALITY=30 INCLUDE_UNPAIRED=false TANDEM_READS=false USE_OQ=true ASSUME_SORTED=true STOP_AFTER=0 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json
 ## htsjdk.samtools.metrics.StringHeader
-# Started on: Thu Mar 19 17:31:03 EDT 2015
```

Comment:
hmmm. I had meant to sort this on the commandline, but it has a new date....I'll upload a sorted version instead...